apps/mull-ui/src/app/pages/create-event/create-event.scss

```
@@ -92,13 +93,26 @@
     padding: 1em 2.5em 1em 1em;
     margin-bottom: 0.275rem;
     width: 100%;
+    margin-top: 0.5rem;
     box-sizing: border-box;
     &.error {
       border-color: $error-color;
     }
   }
 }
 
+.pill-options-container {
+  width: 100%;
+  margin-top: 1rem;
+  .pill-options {
+    width: 100%;
+    margin-top: 0.5rem;
+    .pill-option {
+      width: 100%;
+    }
+  }
+}
+
```

We need remove the left and right border on the second pill. This will do the trick.

```suggestion
.pill-options-container {
  width: 100%;
  margin-top: 1rem;
  .pill-options {
    width: 100%;
    margin-top: 0.5rem;
    button:nth-of-type(2) { // remove border around second pill
      border-right: 0;
      border-left: 0;
    }
    .pill-option {
      width: 100%;
    }
  }
}
```